README.md

```diff
@@ -106,14 +118,14 @@ cls_explainer = SequenceClassificationExplainer("I love you, I like you, I also
 attributions = cls_explainer(class_name="NEGATIVE")
 ```
 
-This still returns a prediction of the **POSITIVE** class
+In this case, `predicted_class_name` still returns a prediction of the **POSITIVE** class, **because...?**
```

Comment:
I'd explain why after the `, because...`

Reply:
Somethink like `because the model has generated the same prediction but nonetheless we are interested in looking at the attributions for the negative class`

Reply:
Great, I'd add that!